integration-tests/bats/remotes.bats

```diff
@@ -1030,6 +1133,12 @@ SQL
     [ "$status" -eq 0 ]
     [[ "$output" =~ "pk" ]] || false
     [[ "$output" =~ "1" ]] || false
+
+    skip # above checkout command should set upstream persisting outside of session
```

Comment:
We store upstream tracking metadata in `repo_state.json` file, and we do not have access to that file from inside the SQL session. As long as, user is in the same session, any changes on upstream tracking works fine, but once out of the session, the changes made inside the session will not be saved since we cannot access the repo_state file

Reply:
Thanks for the extra context. If we don't have a tracking issue for this, could you create one? Seems like something that will confuse/surprise customers and would be great to close this gap before someone falls in it.